bootstrap/src/basis/file.c

```diff
@@ -7,6 +7,7 @@
 #include <unistd.h>
 
 #include <caml/mlvalues.h>
+#include <caml/alloc.h>
```

Comment:
I wish I'd thought to look for this while writing this file. Some of the functions in here dealing with arrays could have been much simpler.